tdrs-backend/tdpservice/users/test/test_api/test_endpoints.py

```diff
@@ -50,10 +50,6 @@ def get_user(self, api_client, user_id):
         """Get a specific user."""
         return api_client.get(f'{self.root_url}{user_id}/')
 
-    def set_profile(self, api_client, profile_data):
```

Comment:
@jtimpe should we remove [this file](tdrs-backend/docs/api/set_profile.md) since we're dropping this?

Reply:
i did remove `docs/api/set_profile.md` as part of this, was there more that i missed?

Reply:
oops, you're correct!